postfix/templates/daemonset.yaml

```diff
@@ -61,6 +63,37 @@ spec:
           {{- if .Values.extraVolumeMounts }}
           {{ toYaml .Values.extraVolumeMounts | nindent 10 }}
           {{- end }}
+      {{- if .Values.metrics.enabled }}
+      - name: exporter
+        image: "{{ .Values.metrics.image.repository }}:{{ .Values.metrics.image.tag }}"
+        imagePullPolicy: {{ .Values.metrics.image.pullPolicy }}
+        {{- with .Values.metrics.command }}
+        command: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        {{- with .Values.metrics.resources }}
+        resources: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        ports:
+        - name: http
+          containerPort: {{ .Values.metrics.port }}
+          protocol: TCP
+        {{- with .Values.metrics.livenessProbe }}
+        livenessProbe: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        {{- with .Values.metrics.readinessProbe }}
+        readinessProbe: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        volumeMounts:
+        - name: spool
+          mountPath: /var/spool/postfix
+          {{- if .Values.persistence.subPath }}
+          subPath: {{ .Values.persistence.subPath }}
+          {{- end }}
+          readOnly: true
+        - name: varlog
+          mountPath: /var/log
+          readOnly: true
```

Comment:
In postfix-exporter to generate metrics from the log file, mount `/var/log`.
However, this feature is off by default.